src/OpenFeature.SDK/OpenFeature.cs

```diff
@@ -81,7 +81,7 @@ public sealed class OpenFeature
         /// Sets the global <see cref="EvaluationContext"/>
         /// </summary>
         /// <param name="context"></param>
-        public void SetContext(EvaluationContext context) => this._evaluationContext = context;
+        public void SetContext(EvaluationContext context) => this._evaluationContext = context ?? new EvaluationContext();

```

Comment:
`??` - features I didn't know about in C#7

Very nice. https://docs.microsoft.com/en-in/dotnet/csharp/language-reference/operators/null-coalescing-operator

Reply:
Now that the dotnet team do a major release every year the language is evolving at a pretty fast rate 👍🏻